BusBoard.ConsoleApp/CliInterface.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusBoard.ConsoleApp
+{
+    class CliInterface
+    {
+        public static void RunUserCommandLoop(TflApi tflApi)
+        {
+            while (true)
+            {
+                Console.WriteLine("\n\"[Bus stop code]\" to list arrivals at that stop");
+                Console.WriteLine("\"Quit\" to quit");
+
+                Console.WriteLine("Please type an option: ");
+
+                var input = Console.ReadLine();
+
+                if (input == "Quit")
+                {
+                    break;
+                }
+                else
```

Comment:
Since you're `break`ing in the `if` above you shouldn't need the `else` which will slightly reduce the nesting 🙂

Reply:
Yep, sorted in f72fd65 🙂